README.md

```diff
@@ -1,7 +1,7 @@
 Grand Central Board for the Apple TV
 ====================================
 
-![image](./README/screenshot.png)
+![image](./README/appletv941.png)
```

Comment:
I think having a screenshot is more attractive - even more since we have more widgets.